### PR TITLE
build(docs-infra): engine "yarn" versions can now be up to 1.16.0

### PR DESCRIPTION
**Pull request by ghost**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
Raises the ceiling on the `yarn` engine version for aio/docs.

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Does not work with yarn 1.16.0

## What is the new behavior?
Works with yarn 1.16.0

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Tested by running the `yarn setup` command to ensure the build process works the same, and then ran `yarn start` to take a quick look at the docs and make sure they're the same.

*I read through [the other PR that was closed](https://github.com/angular/angular/pull/28546). As mentioned, the docs build correctly with yarn 1.16.0.*

### CODE REVIEW

**Comment by rudolfolah** [comment]
I signed it!

**Comment by rudolfolah** [comment]
messed up the commit email address so I think it broke the Google CLA bot, lmk how to fix or if it can be overridden (if the PR is otherwise acceptable).

File: aio/tools/examples/shared/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   },
   "engines": {
     "node": ">=10.9.0 <11.0.0",
-    "yarn": ">=1.10.1 <1.14.0"
+    "yarn": ">=1.10.1 <=1.16.0"
```
Comment (gkalpak) [inline aio/tools/examples/shared/package.json]: Can you also update the min version to match [package.json][1] and [aio/package.json][2]?

[1]: https://github.com/angular/angular/blob/562c0c2434127452fade6c70558f541f2270b11b/package.json#L11
[2]: https://github.com/angular/angular/blob/562c0c2434127452fade6c70558f541f2270b11b/aio/package.json#L75

**Review by gkalpak — CHANGES_REQUESTED** [review]
One minor comment. Otherwise lgtm :+1:

@rudolfolah, the PR is fine, but the CLA issue needs to be sorted. If the problem is that you have associated a different email address with the commi than the one you used to sign the CLA with, you could amend the commit and update the committer info (and force-push to your branch). Alternatively, you could close this PR and open a new one (with the correct committer email).

File: aio/tools/examples/shared/package.json (hunk 1 above)
Comment (rudolfolah) [reply aio/tools/examples/shared/package.json]: updated and force-pushed with the proper email address/author

**Comment by gkalpak** [comment]
The first commit still appears to have the "wrong" committer. Let me fix this for you (but you can find more info on how to change a past commit's author [here][1]).

[1]: https://www.git-tower.com/learn/git/faq/change-author-name-email

**Review by gkalpak — APPROVED** [review]

**Comment by rudolfolah** [comment]
@gkalpak thanks! ah I see I just needed `git commit --amend --author="John Doe <john@doe.org>"`

**Pull request closed without merging**